riklet/src/main.rs

```diff
@@ -4,24 +4,21 @@ mod core;
 mod emitters;
 mod iptables;
 mod network;
+mod runtime;
 mod structs;
 mod traits;
+mod utils;
 
 use crate::core::Riklet;
+use crate::utils::init_logger;
+use anyhow::Result;
+
 use tracing::{event, Level};
-use tracing_subscriber::filter::LevelFilter;
-use tracing_subscriber::EnvFilter;
 
 #[tokio::main]
-async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    tracing_subscriber::fmt()
-        .with_env_filter(
-            EnvFilter::builder()
-                .with_default_directive(LevelFilter::INFO.into())
-                .from_env_lossy()
-                .add_directive("h2=OFF".parse().unwrap()), // disable all events from the `h2` crate
-        )
-        .init();
```

Comment:
The log tracing configuration is missing.

Reply:
no look `init_logger(Some("debug".to_string()))?;` in main